x-pack/plugins/security_solution/public/alerts/pages/detection_engine/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const SINGLE_RULE_ALERTS_TABLE_ID = 'detections-page-rule-details';
```

Comment:
maybe 
```suggestion
export const SINGLE_RULE_ALERTS_TABLE_ID = 'detections-page-rule-details-alerts';
```

Reply:
there is no more detecttion page so can we just do `alerts-rule-details-page`

Reply:
What about `ALERTS_TABLE_ID = 'detections-page-alerts';` ? Its the timeline when you enter to the detection page (detection menu tab).